### DISCUSSION TITLE
Meta-/Extra-data in JSON

### DISCUSSION THREAD

**Discussion by 0xc0de4c0ffee** [namesys-eth/ccip2-eth-resources#16] [Ideas]
\<records>.json is currently formatted as 
```
{
"data":"___data____"
}
```

so I was thinking if it's good idea to add optional meta/extradata for future records manager to read?

```
{
"data": "___data___",
"created" : "timestamp?",
"signer": "0x....",
"domain": "domain.eth" ,.
/// more??
}
```

CCIP callback is only reading "data" value so all extra data is for client/manager and user's reference only.

**Comment by sshmatrix** [comment]
^ added these 4 values to the metadata
